tests/cases/fourslash/refactorConvertParameterPropertyToPropertyDeclaration1.ts

```diff
@@ -0,0 +1,20 @@
+//// class D {
+////     c = 10;
+////     constructor(public /*a*/a/*b*/: string, public readonly b: string) {

```

Comment:
What would happen if the selection were `constructor(/*a*/public a: string, public readonly b: string/*b*/) {`?

Reply:
Unfortunately, Only a will be convert.

Reply:
Please consider adding a test reflecting that behavior.

Reply:
Done. In last commit

Reply:
My mistake. :smile: